tests/h/conftest.py

```diff
@@ -117,7 +117,7 @@ def restart_savepoint(session, transaction):
 @pytest.fixture(scope='session', autouse=True)
 def deform():
     """Allow tests that use deform to find our custom templates."""
-    form.init()
+    form.includeme(config=mock.Mock())
```

Comment:
I've corrected the docstring, which I think was slightly off. What calling this function actually does is register our custom deform renderer so that it gets used instead of deform's default chameleon template renderer (which crashes if the tests use it), so yes it means that deform can "find our custom templates" but what it really means is that deform will use our renderer not its default one.

I haven't changed the name of this fixture but I think `deform` is an odd name for it, it's not for example patching deform with a mock and returning the mock, a better name for it would probably be `DeformJinja2Renderer`.

It seems to be fine to pass in a mock in place of the configurator. Which will mean that our deform jinja2 renderer ends up with a mock object as its jinja2 env, mock objects as templates, `template.render()` will do nothing and return a mock ... Is it worth looking into passing in a real test configurator instead, so that our templates actually get rendered? I suppose they were actually being rendered before.